test/parallel/test-fs-promises-file-handle-read.js

```diff
@@ -47,28 +48,27 @@ async function validateRead(data, file) {
   await streamFileHandle.close();
 }
 
-async function validateLargeRead() {
+async function validateLargeRead(options) {
   // Reading beyond file length (3 in this case) should return no data.
   // This is a test for a bug where reads > uint32 would return data
   // from the current position in the file.
   const filePath = fixtures.path('x.txt');
   const fileHandle = await open(filePath, 'r');
   const pos = 0xffffffff + 1; // max-uint32 + 1
-  const readHandle = await read(fileHandle, Buffer.alloc(1), 0, 1, pos);
+  const readHandle =
+    await read(fileHandle, Buffer.alloc(1), 0, 1, pos, options);
 
   assert.strictEqual(readHandle.bytesRead, 0);
 }
 
-let useConf = false;
 
 (async function() {
-  for (const value of [false, true]) {
-    tmpdir.refresh();
-    useConf = value;
-
-    await validateRead('Hello world', 'tmp-read-file.txt')
-      .then(validateRead('', 'tmp-read-empty-file.txt'))
-      .then(validateLargeRead)
+  tmpdir.refresh();
+  await validateRead('Hello world', 'read-file', { useConf: false })
+      .then(validateRead('', 'read-empty-file', { useConf: false }))
+      .then(validateRead('Hello world', 'read-file-conf', { useConf: true }))
+      .then(validateRead('', 'read-empty-file-conf', { useConf: true }))
+      .then(validateLargeRead({ useConf: false }))
+      .then(validateLargeRead({ useConf: true }))
       .then(common.mustCall());
```

Comment:
You should either get rid of the wrapping async function or replace `.then` with `await`s, the current implementation is a bit silly 😝 
```suggestion
  await validateRead('Hello world', 'read-file', { useConf: false });
  await validateRead('', 'read-empty-file', { useConf: false });
  await validateRead('Hello world', 'read-file-conf', { useConf: true });
  await validateRead('', 'read-empty-file-conf', { useConf: true });
  await validateLargeRead({ useConf: false });
  await validateLargeRead({ useConf: true });
```